docs/source/customizations/coprocessor.mdx

```diff
@@ -577,17 +606,40 @@ This value is present only for coprocessor requests from the router's `SubgraphS
 </td>
 <td>
 
-The URL of the subgraph that this coprocessor request pertains to.
+The URL that this coprocessor request pertains to.
 
-This value is present only for coprocessor requests from the router's `SubgraphService`.
+At the `RouterService` stage, this is the path the resquest was made to.
```

Comment:
🤔  Is the "path" distinction here really true?  Put another way, `uri` is just a `path` at the `RouterService` level but a full URL on the `SubgraphService`? 

If so, not only does that seem confusingly different, but that would also go against the definition of a URI, wouldn't it? (Doesn't "URI" demand that the there be a scheme on the front of it? Like, that's the part that makes it "Uniform", right?)

Reply:
hmmm indeed, let's make it `path` right now before it becomes a breaking change!